evalml/tests/automl_tests/test_engine_base.py

```diff
@@ -103,7 +103,7 @@ def test_train_and_score_pipelines_error(
     assert "yeet" in caplog.text
 
 
-@patch("evalml.automl.engine.engine_base.split_data")
+@patch("evalml.automl.utils.split_data")
```

Comment:
The logic isn't changing, so I didn't add any tests that specifically test these utils on their own, but I'd be open to adding some if that's a necessary part of pulling this logic out into public utils